src/SourceLink.Tools/Microsoft.SourceLink.Tools.Package.csproj

```diff
@@ -1,19 +1,12 @@
-<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
We do want this project to compile the source. This is to validate that the source package code is correct and builds against the listed TFMs.

Reply:
Makes sense. Reverted the change.